app/iot/devices.py

```diff
@@ -1,64 +1,64 @@
-import time
+import asyncio
 
-from .message import MessageType
+from app.iot.message import MessageType
```

Comment:
Ensure that the 'MessageType' class from 'app.iot.message' has a 'name' attribute, as it is used in the 'send_message' method.